google/cloud/firestore_admin_v1/services/firestore_admin/async_client.py

```diff
@@ -273,9 +273,9 @@ async def list_indexes(
                 maximum=60.0,
                 multiplier=1.3,
                 predicate=retries.if_exception_type(
+                    exceptions.DeadlineExceeded,
                     exceptions.InternalServerError,
                     exceptions.ServiceUnavailable,
-                    exceptions.DeadlineExceeded,
```

Comment:
@busunkim96, @crwilcox, @software-dov Here we are again re-ordering exception types.  Note that some changes below are *not* alpha-sorted.

Reply:
Opened https://github.com/googleapis/gapic-generator-python/issues/617 to track.